docs/content/en/docs/implementing/evaluations.md

```diff
@@ -3,3 +3,67 @@ title: Evaluations
 description: Understand Keptn evaluations and how to use them
 weight: 150
 ---
+A `KeptnEvaluation` checks whether a `KeptnMetric` meets a defined target value.
+A `KeptnEvaluation` resource is defined in a
+[KeptnEvaluationDefinition](../yaml-crd-ref/evaluationdefinition.md)
+yaml file.
+The example
+[app-pre-deploy-eval.yaml](https://github.com/keptn/lifecycle-toolkit/blob/main/examples/sample-app/version-3/app-pre-deploy-eval.yaml)
+file specifies the `app-pre-deploy-eval-2` evaluation as follows:
+{{< embed path="/examples/sample-app/version-3/app-pre-deploy-eval.yaml" >}}
+
+The `evaluationTarget` is set to be `>1`,
+so this evaluation ensures that more than 1 CPU is available
+before the workload or application is deployed.
+
+The example
+[metric.yaml](https://github.com/keptn/lifecycle-toolkit/blob/main/examples/sample-app/base/metric.yaml)
+file defines the
+[KeptnMetric](../yaml-crd-ref/metric.md) resource
+that is named  `available-cpus`:
+{{< embed path="/examples/sample-app/base/metric.yaml" >}}
+
+To run an evaluation on one of your
+[Workloads](https://kubernetes.io/docs/concepts/workloads/)
+([Deployments](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/),
+[StatefulSets](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/),
+[DaemonSets](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/),
+or
+[ReplicaSets](https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/),
+you must:
+
+* Annotate your [Workloads](https://kubernetes.io/docs/concepts/workloads/)
+  [Deployments](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/),
+  [StatefulSets](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/),
+  and
+  [DaemonSets](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/)
+  to include each evaluation and task you want to run
+  pre- and post-deployment for the specific workloads.
+* Manually edit all
+  [KeptnApp](../yaml-crd-ref/app.md) resources
+  to specify the evaluations to be run
+  pre- and post-deployment for the `KeptnApp` itself.
+
+See [Pre- and post-deployment checks](../implementing/integrate/#pre--and-post-deployment-checks)
+for details.
+
+Note the following:
+
+* One `KeptnEvaluationDefinition` resource can include
+  multiple `objective` fields that reference additional metrics.
+  In this example, you might want to also query
+  available memory, disk space, and other resources
+  that are required for the deployment.
+* The `KeptnMetric` resources that are referenced
+  in a `KeptnMetricDefinition` resource
+  can be defined on different namespaces in the cluster.
+* All `KeptnEvaluation` resources
+  that are defined by `KeptnEvaluationDefinition` resources at the same level
+  (either pre-deployment or post-deployment)
+  execute in parallel.
```

Comment:
There is always only one evaluation running for a specific pre/post task, never more. Also KeptnEvaluation is not defined by KeptnEvaluationDefinition, but it's created according to the information that are defined in the KeptnEvaluationDefinition. In example you can think of it like a Definition describes what action needs to be done and the evaluation describes how this action was completed and with what kind of result

Reply:
Thanks for clarifying this -- I did misunderstand it, which made for all sorts of errors!  See if the new version is better.

Reply:
@odubajDT maybe we need to clarify this "There is always only one evaluation running for a specific pre/post task, never more." users can specify a list of evaluation definitions to run as a list of strings in the app or workload level

Reply:
This was clarified in a slack conversation.  One `KeptnEvaluationDefinition` can include multiple objectives (which will be evaluated sequentially)  but you can also run multiple `KeptnEvaluationDefinition` resources for the same stage and they will run in parallel.  I will correct the documentation.